### DISCUSSION TITLE
유니티 인스펙터에 클래스로 만든 타입이 표시되도록 할수 있을까요?

### DISCUSSION THREAD

**Discussion by pda017** [naverz/zepeto-studio-kor#230] [5. Q&A]
c# 에선 class 위에 [system.serializable] 이거 붙이면 됬었는데. 타입스크립트에선 어떻게 해야 하나요?

**Comment by x4cf** [comment]
지금은 인스펙터에 노출시킬 방법이 없어보입니다.
저는 인스펙터에 오브젝트로 받아와서 겟 컴포넌트로 클래스를 받아옵니다

**Reply by pda017** [reply]
> ㅠㅠ... 에디터 확장이나 인스펙터를 제대로 활용할수 없으면 유니티를 쓰는 의미가 크게 있나 싶긴 하네요 ㅠ...
제가 너무 편하게 코딩해온건지 ㅠㅠ...

**Comment by hyeonjin-jo** [comment]
안녕하세요.

현재는 해당 기능을 제공하지 않고 있으나 내부적으로 논의되고 있는 부분인 점 참고 부탁드립니다.

감사합니다.
